src/error.rs

```diff
@@ -125,3 +126,24 @@ impl From<JSONError> for Error {
     }
   }
 }
+
+impl From<YAMLError> for Error {
+  fn from(error: YAMLError) -> Self {
+    match &error {
+      &YAMLError::Custom(ref message) => {
+        Error {
+          code: ErrorCode::BadRequest,
+          message: message.clone(),
+          hint: Some("Fix your YAML syntax.".to_string())
```

Comment:
Bad YMAL Synatx
Unparsable YMAL Synatx
There was an error with the YMAL Synatx

Something along those lines?